app/views/layouts/application.html.erb

```diff
@@ -19,6 +19,8 @@
     <% if noindex_page? %>
       <meta name="robots" content="noindex" />
     <% end %>
+      <%# theme-color is from app/assets/stylesheets/petitions/_colours.scss:$house-of-commons-green %>
```

Comment:
I don't think the comment is really necessary - can you remove it please.

Reply:
sure